api/proto/config/appconfig.proto

```diff
@@ -26,6 +26,47 @@ enum MetaDataType {
   MetaDataDriveMultipart = 3; // Process multipart MIME for application
 }
 
+// The information about a snapshot
+message AvailableSnapshot {
+  string id = 1;
```

Comment:
Is this a uuid, a timestamp, or something else?

Reply:
I would use UUID.

Reply:
But that assumption (and the responses to all other comments) need to be added to the comments in the proto file. Think of the proto file comments as the contract between EVE and the various controller implementors.

As it matters in this case, because below there are comments/questions whether the id is or includes a timestamp.

Reply:
I tried to explain my vision of the ID field here: https://github.com/lf-edge/eve/pull/3018#discussion_r1099400415
If you agree, I'll update the comment with this info.

Reply:
Good start and right level of detail. But see below about UUID vs UUID+something.

Reply:
Ok, let's figure out a good format for the ID, and I'll put it in the comment then.